angular/PdfViewer/Text-Markup-Annotation.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Text markup annotation of PDF viewer component.
-description: Text markup annotation of PDF viewer component.
+title: Text markup annotation in Angular PDF viewer component | Syncfusion
+description: Learn here about Text markup annotation in Syncfusion Essential Angular PDF Viewer component, its elements, and more.
 platform: Angular
 control: PDF viewer
 documentation: ug
 keywords: ejPdfViewer, PDF viewer, js pdfviewer
 ---
 
-## Text Markup Annotation
+# Text Markup Annotation in Angular PDF Viewer
 
 PDF viewer control supports adding text markup annotations in the PDF documents. The component also renders the existing text markup annotations from the PDF document when the document is loaded in it.
 
 The following screenshot shows PDF viewer with the PDF document containing text markup annotations.
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img1.png)
+![Adding text markup in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img1.png)
```

Comment:
It should be Text markup annotations in Angular PDF Viewer

Reply:
Updated